editor/src/clj/editor/app_view.clj

```diff
@@ -630,32 +630,6 @@
     (build-errors-view/update-build-errors build-errors-view error-value)
     (show-build-errors! main-scene tool-tab-pane)))
 
-(def ^:private remote-log-pump-thread (atom nil))
```

Comment:
Console relater code moved to `console/` to be able to use it from both `targets` and `app-view`